blueutil.m

```diff
@@ -301,6 +420,81 @@ bool parse_signed_long_arg(char *arg, long *number) {
   }
 }
 
+NSArray *get_paired_devices_subprocess() {
+  NSTask *task = [[NSTask alloc] init];
+  task.launchPath = @"/usr/sbin/system_profiler";
+  task.arguments = @[@"SPBluetoothDataType", @"-xml"];
+
+  NSPipe *pipe = [NSPipe pipe];
+  task.standardOutput = pipe;
+  task.standardError = [NSPipe pipe];
```

Comment:
Probably better to not hide standard error output, as it will be harder to debug when something goes wrong